openedx/core/djangoapps/course_date_signals/models.py

```diff
@@ -0,0 +1,11 @@
+from openedx.core.djangoapps.config_model_utils.models import StackedConfigurationModel
+
+
+class SelfPacedRelativeDatesConfig(StackedConfigurationModel):
```

Comment:
Is this like, the waffle-ish solution we're using for rollout here, or is the intention that once we finish this feature, we'd still turn it on or off for special courses?

Reply:
@sstack22, do we need to support long-term exceptions for particular courses or orgs (not just for individual course runs)?

Reply:
Coming back around: I checked w/ @sstack22, and we will likely need to make long-term exceptions for some non-linear courses.